utils/build/docker/java/parametric/libraryversion.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
There is a small convention, could you rename this file to `install_ddtrace.sh` ?

Reply:
@cbeauchesne Even is the selection of the tracer to run is done in run.sh? 
libraryversion.sh only stores the tracer version in SYSTEM_TESTS_LIBRARY_VERSION

Reply:
This was written during the zoom and install_ddtrace.sh was deleted